### DISCUSSION TITLE
Storage emulator using filesystem for persistence

### DISCUSSION THREAD

**Discussion by shaileshkm** [googleapis/google-cloud-cpp#6207] [Q&A] [answered]
Hi,
I'm experimenting with Storage SDK emulator (google/cloud/storage/emulator). I believe it maintains buckets/objects in memory. Any idea if it already supports filesystem persistence?
I tried searching in Issues but did not find any thread on it.

regards,
_shailesh

**Comment by coryan** [comment, marked as answer]
No support for persisting data.  Nor do we have any plans to add such support.
